JAVA/2.Basic Syntax/Basic Syntax.md

```diff
@@ -0,0 +1,60 @@
+
+# Java Basic Syntax
+Here are some basic syntax elements of the Java programming language:
+
+
```

Comment:
Remove the extra lines from the whole file. There must be only one line space if needed.